src/components/chat-item/chat-prompt-input.ts

```diff
@@ -458,24 +458,43 @@ export class ChatPromptInput {
   };
 
   private readonly sendPrompt = (): void => {
+    const quickPickItems = MynahUITabsStore.getInstance().getTabDataStore(this.props.tabId).getValue('quickActionCommands') as QuickActionCommandGroup[];
     const currentInputValue = this.promptTextInput.getTextInputValue();
     if (currentInputValue.trim() !== '' || this.selectedCommand.trim() !== '') {
+      let selectedCommand = this.selectedCommand;
+
+      // Catching cases where user could send a prompt with quick action command but the command is not be selected correctly
+      if (selectedCommand === '') {
+        for (const quickPickItem of quickPickItems) {
+          if (selectedCommand !== '') break;
+          const matchedCommand = quickPickItem.commands.find((item) => currentInputValue.startsWith(item.command));
```

Comment:
if the the commend starts with a space " /dev" it still be activated?

Reply:
nope, it won't

Reply:
Do we want to trigger that if this happens? We can potentially trim the space before command so it triggers in the same way.

Reply:
I don't think that's an intended behaviour, if you try right now, ` /` does not trigger quick action.

Reply:
Yes, I know that currently does not trigger quick action. 

However, I don't see another intent for the user rather than open a new tab with the command even if add mistakenly a space before. I'm wondering if we shouldn't fix this also as part of the customer feedback.